source/_integrations/unifiprotect.markdown

```diff
@@ -337,3 +345,7 @@ Similarly, a `502 Bad Gateway` also means that your UniFi Protect application ma
 ```log
 pyunifiprotect.NvrError: Fetching Camera List failed: 404 - Reason: Not Found
 ```
+
+### Cannot Remove Ignored Device
+
+Due to an [existing frontend bug](https://github.com/home-assistant/frontend/issues/9506), you may have to set the value to a single space (`" "` without quotes) to clear out all of the existing ignored MAC addresses.
```

Comment:
We don't document bugs. For that, we have an issue tracker, and we link to known issues in different places.

```suggestion
```

Reply:
You just made the argument what I consider simple is not simple for others. Understanding the Home Assistant Github and issue structure maybe be simple for you because you spend all day in Github with Home Assistant, but from an outside user, that is likely not a developer, they are just going be pissed off that they cannot remove an ignored device. 

This is going to be a very common roadblock for anyone trying to remove an ignored device and I think it need to be documented until it is fixed. If you do not want it here, go fix the issue.

Reply:
> If you do not want it here, go fix the issue.

Feel free to help out. Until that time, this needs to be removed from this documentation PR.